packages/js/components/src/table/index.tsx

```diff
@@ -184,6 +185,11 @@ const TableCard: React.VFC< TableCardProps > = ( {
 			{ /* Ignoring the error to make it backward compatible for now. */ }
 			{ /* @ts-expect-error: size must be one of small, medium, largel, xSmall, extraSmall. */ }
 			<CardBody size={ null }>
+				{ tablePreface && (
+					<div className="woocommerce-table__table-preface">
```

Comment:
Should we add a `x` button to the preface so that the user can remove it after observing it? That can then be retained in some state to ensure we don't keep showing the same section to the user. 

That could also be an optional argument to this component so that the dev can decide whether they want to make it "closeable" or not. WDYT? 🤔

Reply:
> Should we add a x button to the preface so that the user can remove it after observing it?

That's a good point and it would align well with our use case. But my intention was to keep it generic and implement a "slot" in a Vue js style if you will. So the idea is to allow passing a component and render it before the table, without adding more functionality. This way we keep the component lean, and leave the responsibility of rendering the preface or not outside the TableCard component.

That said, I wonder if this pull request still makes sense, now that we decided to go with a different approach to implement the merchant notice.I'd love to get @rogermattic's input on this, especially to understand the vision from a design perspective.